core/controller/src/main/scala/org/apache/openwhisk/core/controller/Controller.scala

```diff
@@ -230,6 +233,10 @@ object Controller {
   def main(args: Array[String]): Unit = {
     implicit val actorSystem = ActorSystem("controller-actor-system")
     implicit val logger = new AkkaLogging(akka.event.Logging.getLogger(actorSystem, this))
+    if (useClusterBootstrap) {
+      AkkaManagement(actorSystem).start()
+      ClusterBootstrap(actorSystem).start()
+    }
```

Comment:
As far as i know, ShardingContainerPoolBalancer already has relative logic.
![image](https://user-images.githubusercontent.com/11749867/180123883-40e882ea-9c7f-4d1f-aa05-82c35ff52217.png)
Since ShardingContainerPoolBalancer's codes and these codes locates in same place, does it repeat?

Currently, we have ShardingContainerBalancer and FPCPoolBalancer, seems can delete the Sharding one, just keep here?